conf/logback.xml

```diff
@@ -4,13 +4,13 @@
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
```

Comment:
Would be nice to remove the debug asset request logs as they clog up the channel. Perhaps we could default local env to INFO level?